commpy/examples/conv_encode_decode.py

```diff
@@ -1,57 +0,0 @@
-
-# Authors: Veeresh Taranalli <veeresht@gmail.com>
-# License: BSD 3-Clause
-
-import numpy as np
```

Comment:
@BastienTr Was there a reason why you deleted this example? I didn't find another example for this. It could be very helpful for users to look at examples before they start using CommPy.

Reply:
I don't remember exactly... At the moment, I think that it was because it was not written following the current philosophy of the library but there must be something else. 

Anyway, I'll provide a new example in the next days so that the problem is solved :)

Reply:
I haded an example using the old and simple style as well as the CommPy style.

Reply:
Thank you @BastienTr  :+1: